x/bank/Cargo.toml

```diff
@@ -5,10 +5,11 @@ edition = "2021"
 
 [dependencies]
 #local
-gears = { path = "../../gears", features = [ "cli", "export", "xmods"] }
+gears = { path = "../../gears", features = ["cli", "export", "xmods"] }
 
 #newtypes
 bytes = { workspace = true }
+ibc-proto = { workspace = true } # TODO: should we import through gears via core-types?
```

Comment:
I don't think so

Reply:
If some structure will be needed in other crate then we need to create local copy of this type or re-export type